### ISSUE TITLE
citationType field has inconsistent casing

### ISSUE THREAD

**Issue by MattBlissett** [gbif/content-crawler#52]
Untyped enumerations mean incomplete search results:

* https://api.gbif.org/v1/literature/search?limit=0&facet=citationType
* https://api.gbif.org/v1/literature/search?citationType=generic
* https://api.gbif.org/v1/literature/search?citationType=Generic

Maybe this should be an enumeration?

**Comment by dnoesgaard** [comment]
I probably had a typo somewhere at some point. I've corrected the "Generic" ones to "generic". But it would be nice if the crawler used an enum.

**Comment by timrobertson100** [comment]
A [vocabulary](https://registry.gbif.org/vocabulary/search) might be appropriate, where the content crawler downloaded the full vocabulary on each run, allowing us to expand and add synonyms without code changes.

**Comment by dnoesgaard** [comment]
That sounds interesting—and like something that could work for other fields as well, e.g. `topics`

**Comment by timrobertson100** [comment]
> and like something that could work for other fields as well, e.g. topics

Yes, plus it provides the definition of what the terms actually mean, should anyone be confused (e.g. future versions of ourselves).

**Comment by MattBlissett** [comment]
Note three other fields already have enumerations, LiteratureRelevance, LiteratureTopic and LiteratureType: https://gbif.github.io/gbif-api/apidocs/org/gbif/api/model/literature/package-frame.html
